pkg/loop/internal/pb/capabilities_registry.proto

```diff
@@ -22,6 +23,22 @@ enum ExecuteAPIType {
   EXECUTE_API_TYPE_CALLBACK = 2;
 }
 
+
+// DON has arguments for [github.com/smartcontractkit/chainlink-common/pkg/capabilities.DON].
+message DON {
+  string id = 1;
+  repeated bytes members = 2; // [membern][32]byte
```

Comment:
typo in `membern`?  [][32]byte?

Reply:
not a typo, but if its not obvious i should change it to `n` instead, thoughts? its supposed to mean the nth member